readme/apps/debugging.md

```diff
@@ -2,6 +2,16 @@
 
 It is possible to get the apps to display or log more information that might help debug various issues.
 
+:::note
+
+Before posting a full log to GitHub or the forum, be aware that it may contain (but isn't limited to):
+- Note/notebook/tag titles
+- Timestamps
+- Sync target information 
+- Encrypted copies of master keys
+
+:::
+
 ## Desktop application
```

Comment:
Coud you please remove this. We already struggle to get people to share their log and with a warning like we can be sure nobody will. We made a lot of efforts to ensure that the log doesn't contain sensitive data and we'll keep making such efforts, but it's not a good idea to discourage people from sharing information we need to support them.

Reply:
Removed!